user-service/middleware/auth.js

```diff
@@ -0,0 +1,27 @@
+import jwt from 'jsonwebtoken';
+import 'dotenv/config';
+import { ormCreateBlacklist as _createBlacklist } from '../model/blacklist-orm.js';
+import { ormCheckTokenBlacklist as _checkTokenBlacklist } from '../model/blacklist-orm.js';
+
+export async function authenticateJwt(req, res, next) {
+    try {
+        const authorization = req.get('authorization');
+        // Check if token exists
+        if (!authorization || !authorization.toLowerCase().startsWith('bearer ')) {
+            return res.status(403).send({message: "A token is required for authentication, please log in again!"});
```

Comment:
Should this be 401 under invalid credentials?

https://stackoverflow.com/questions/3297048/403-forbidden-vs-401-unauthorized-http-responses

Reply:
Solved